gp2gp-translator/src/test/resources/xml/Encounter/availability_time_encounter_period.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just saying that for testing purposes, the <?xml> line isn't needed for Jaxb to correctly unmarshall it